sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/query.sql

```diff
@@ -154,7 +154,7 @@ WITH base AS (
         ) AS value
     ) AS keyed_hist_sums,
   FROM
-    `moz-fx-data-shared-prod`.telemetry_stable.main_v4
+    `moz-fx-data-shared-prod`.telemetry_stable.main_v5
```

Comment:
Looks like the v5 schema isn't identical to v4 in terms of record field order (v5 appears to have fields in alphabetical order, while v4 has initial fields in alphabetical order followed by other fields in the order they were added to the table).  This is causing an error when calling `udf_js.main_summary_active_addons()` in this ETL.

Reply:
updated the udf to use main_v5 column order